2-environments/README.md

```diff
@@ -30,10 +30,25 @@ The purpose of this step is to set up dev, nonprod, and prod environments within
 
 
 ### Run terraform locally
-1. Change into 2-environments folder
-1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap. Copy terraform.tfvars into each of the folders within the envs/ folder.
-1. Within each envs/ folder, update backend.tf with your bucket name from the bootstrap step.
-1. In sequence, change into each folder within the envs/ folder and run the following commands:
-    1. Run `terraform init`
-    1. Run `terraform plan` and review output
-    1. Run `terraform apply`
+1. Change into 2-environments folder.
+1. Run `cp ../build/tf-wrapper.sh .`
+1. Run `chmod 755 ./tf-wrapper.sh`
+1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Update backend.tf with your bucket from bootstrap. You can run
+```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```.
+You can run in the 0-bootstap folder `terraform output gcs_bucket_tfstate` to obtain the bucket name.
+
+We will now deploy each of our environments(dev/prod/stage) using this script.
+When checked into a VCS, each environment corresponds to a branch and only the corresponding environment is applied.
```

Comment:
I think this part cloud be more direct saying something like: 
> when using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for `2-environments` step and only the corresponding environment is applied.

binding the branch name to the file system folder is done by the configuration on Cloud Build or Jenkins